examples/python/nixl_gds_example.py

```diff
@@ -29,6 +30,16 @@
         print("Please specify file path in argv")
         exit(0)
 
+    test_file = sys.argv[1]
+
+    # Check if file exists and create/open it
+    if not file_exists(test_file):
+        print(f"Creating new file: {test_file}")
```

Comment:
pls use nixl absl-based logger or remove this

Reply:
Will change them.

Reply:
Actually absl logger is only for C++ and not python, so cannot use them here.

Reply:
@yosefe is it possbile to remove this change request since it is not possible to use absl for python currently?